streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -477,7 +491,7 @@ public void produce(String topic) throws Exception {
     private void produce(String topic, int valueSizeBytes, String clientId, int numRecords, boolean sequential,
                          int upperRange, boolean printStats) throws Exception {
 
-        processedRecords = 0;
+        processedRecords.set(0);
```

Comment:
maybe make a method for resetting `processedRecords` and `processedBytes` to 0? I can see the same thing done it at least 3 places